data/categories.txt

```diff
@@ -14991,3 +14991,178 @@ uk:Електростанція
 vi:Nhà máy điện
 zh-Hans:发电厂
 zh-Hant:發電廠
+
+Generator|@generator
```

Comment:
Did you read the comment at the top of the file?

Reply:
hmm, yes i think i have mistaken somewhere sorry from my side...